src/Stack/Types/Build.hs

```diff
@@ -357,7 +358,11 @@ instance NFData BuildCache
 instance Store BuildCache
 
 buildCacheVC :: VersionConfig BuildCache
+#if __GLASGOW_HASKELL__ >= 802
+buildCacheVC = storeVersionConfig "build-v1" "IwLXp-Y9VptuI0_MD15kAGNLLZQ="
```

Comment:
I don't know why these are changed, can dependencies affect these?

Reply:
ping @mgsloan

Reply:
When the hash mismatches, it tells you how to diff the schema.  It can indeed be based on dependencies.  If it is just a renaming / relocation, can add it to here - https://github.com/commercialhaskell/stack/blob/master/src/Data/Store/VersionTagged.hs#L100 .

Reply:
@mgsloan is there a way to get schema for diffing? I don't see any helper in http://hackage.haskell.org/package/store-0.4.3.2/docs/Data-Store-Version.html for that

Reply:
@phadej The TH error message should have the diff invocation, the schema files get stored in the dist directory.